Makefile

```diff
@@ -175,3 +186,28 @@ lint-python:
 	pip install -q -r python/test-requirements.txt
 	LC_ALL=C.UTF-8 LANG=C.UTF-8 black --config python/pyproject.toml python/
 	cd python && pylint -j4 `find . -maxdepth 1 -mindepth 1 -type d -o -name "*.py"`
+
+.PHONY: devstack
+devstack:
+	docker build -t devstack:latest test-services/devstack
+	docker run -d --privileged \
+		--name devstack \
+		-v /lib/modules:/lib/modules:ro \
+		$(docker_env) devstack:latest
+	docker exec devstack start-devstack.sh
+	docker exec devstack su - stack -c 'cd /opt/stack/devstack && ./unstack.sh'
+	docker export devstack > /tmp/devstack.tar
+	docker import /tmp/devstack.tar quay.io/signalfx/devstack:latest
+	docker rm -fv devstack
+	rm -f /tmp/devstack.tar
+
+.PHONY: run-devstack
+run-devstack:
+	docker run -d --privileged \
```

Comment:
Same thing as above.

Reply:
I would actually just remove this make target and have it in a script if it ever needs to be run.